packages/veritone-react-common/src/components/FormBuilder/FormBuilder.js

```diff
@@ -31,8 +31,13 @@ function FormBuilderComponent({
   swapBlock,
   updateBlock,
   removeBlock,
-  selectBlock
+  selectBlock,
+  classes,
+  helpLink
 }) {
+
+  const styles = useStyles({});
+
   const [isPreview, setIsPreview] = React.useState(false);
 
   const [, drop] = useDrop({
```

Comment:
What is the purpose of this? `,drop`

Reply:
drop is one of the function in which it will support the drop target of element